contribute/architecture/README.md

```diff
@@ -6,4 +6,5 @@ Learn more about the backend architecture:
 
 - Part 1: [Services](services.md) 
 - Part 2: [Communication](communication.md)
+- Part 2: [State](state.md)
```

Comment:
```suggestion
- Part 3: [State](state.md)
```

Maybe consider renaming `State` to `Persistence` or `Database` or `Database Persistence`? Same applies to name of the file and the page title.